### DISCUSSION TITLE
Tying sandboxie-plus to the second gpu and size limit of files

### DISCUSSION THREAD

**Discussion by Zeond1987** [sandboxie-plus/Sandboxie#3357] [Q&A Feedback]
Hi, I have a couple of questions.

1. How do you make sandboxie use a second gpu for rendering graphics in programs?
2. How do you change the size limit of files and programs that sandboxie is allowed to store in it's sandbox?

**Comment by bastik-1001** [comment]
1. Programs should be able to detect the second GPU and able to select it.
2. Unless you are talking about RAM disks, there is no limitation, other than the storage-device itself. If you are talking about the file-copy limit, which cares about how big files can be before they are too big to be migrated, that can be changed at File Options > [File Migration]

**Comment by Zeond1987** [comment]
1. What if the program isn't sophisticated enough to detect it? How do I force the sandboxed program to use it?

**Reply by bastik-1001** [reply]
As far as I know, you should be able to select which GPU is going to be used in the software that the graphics cards vendor supplies. Assuming that you can launch such a tool in the same sandbox, you should be able to make an application use GPU A in one box and GPU B in another box.

Right now, I don't think Sandboxie itself is doing anything when it comes to GPU selection.

**Comment by offhub** [comment]
https://www.intel.com/content/www/us/en/support/articles/000090168/graphics.html
